### ISSUE TITLE
Cannot find module `node_modules/@orama/plugin-data-persistence/dist/server-commonjs.cjs`

### ISSUE THREAD

**Issue by moltar** [oramasearch/orama#421]
**Describe the bug**

Switching to `/server` exports per docs:

https://docs.oramasearch.com/plugins/plugin-data-persistence

```ts
import { persistToFile } from '@orama/plugin-data-persistence/server'
```

Getting the error:

```
Error: Cannot find module '.../node_modules/@orama/plugin-data-persistence/dist/server-commonjs.cjs'
```

Upon `dist` inspection, I don't see `server-commonjs.cjs` there (only `server-commonjs.js`).

```
❯ ls node_modules/@orama/plugin-data-persistence/dist/   
      
commonjs.cjs           errors.js              index.js.map           server.d.ts            types.js               utils.js.map
commonjs.cjs.map       errors.js.map          server-commonjs.d.cts  server.js              types.js.map
commonjs.d.cts         index.d.ts             server-commonjs.js     server.js.map          utils.d.ts
errors.d.ts            index.js               server-commonjs.js.map types.d.ts             utils.js
```

**To Reproduce**

1. Install the module.
2. Try to import as documented.

**Expected behavior**

Imports should work as documented.

**Screenshots**

N/A

**Desktop (please complete the following information):**

N/A

**Smartphone (please complete the following information):**

N/A

**Additional context**

Applying the following patch fixes the runtime, but TypeScript type check still fails:

```patch
diff --git a/package.json b/package.json

--- a/package.json
+++ b/package.json
@@ -15,8 +15,8 @@
     "./server": {
       "types": "./dist/server.d.ts",
       "import": "./dist/server.js",
-      "require": "./dist/server-commonjs.cjs",
-      "node": "./dist/server-commonjs.cjs"
+      "require": "./dist/server-commonjs.js",
+      "node": "./dist/server-commonjs.js"
     }
   },
   "types": "./dist/index.d.ts",

```

**Comment by micheleriva** [comment]
Hi @moltar, thanks for opening this and for providing such a great context. We're planning a release for tomorrow, and we'll make sure to include a fix for this.

@ShogunPanda could you please take care of this for tomorrow's release?

**Comment by thaoula** [comment]
Hi @micheleriva, 

Has this been released? I cannot seem to able to reference:
 import { persistToFile } from '@orama/plugin-data-persistence/server' either.

**Comment by micheleriva** [comment]
Hi, not yet. I'll try to release today

**Comment by micheleriva** [comment]
Hi @moltar,  @thaoula, this has been released with v1.0.8. Please feel free to reopen if you notice any other issue 🙏
